Geometry_Engine/Query/Bounds.cs

```diff
@@ -255,10 +255,9 @@ public static BoundingBox Bounds(this Line line)
 
         /***************************************************/
 
-        [NotImplemented]
         public static BoundingBox Bounds(this NurbsCurve curve)
         {
-            throw new NotImplementedException();
+            return curve.ControlPoints.Bounds(); // TODO: Bounds(NurbsCurve) incorrect
```

Comment:
Why is the comment left in this case? If there is a bug, please raise an issue and either refer to it or delete the comment altogether.